lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KBVGatewayFactory.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.di.ipv.cri.kbv.api.gateway;
+
+import com.experian.uk.schema.experian.identityiq.services.webservice.IdentityIQWebService;
+import com.experian.uk.wasp.TokenService;
+import uk.gov.di.ipv.cri.common.library.service.ConfigurationService;
+import uk.gov.di.ipv.cri.kbv.api.security.Base64TokenCacheLoader;
+import uk.gov.di.ipv.cri.kbv.api.security.HeaderHandler;
+import uk.gov.di.ipv.cri.kbv.api.security.HeaderHandlerResolver;
+import uk.gov.di.ipv.cri.kbv.api.security.KBVClientFactory;
+import uk.gov.di.ipv.cri.kbv.api.security.SoapToken;
+
+public class KBVGatewayFactory {
+    private final KBVGateway kbvGateway;
+
+    public KBVGatewayFactory(ConfigurationService configurationService) {
+        HeaderHandler headerHandler =
+                new HeaderHandler(
+                        new Base64TokenCacheLoader(
+                                new SoapToken(
+                                        "GDS DI",
+                                        true,
+                                        new TokenService(),
+                                        configurationService.getSecretValue(
+                                                "experian/iiq-wasp-service"))));
+
+        new KeyStoreLoader(configurationService).load();
```

Comment:
I deliberated about where to invoke the `load()` method and after some experimentation, I concluded that the KBVGatewayFactory is responsible for the creation of the KBVGateway (and a KBVGateway cannot be instantiated without invoking the factory) and therefore the pre-requisite SSL setup work can happen here.
This may change in the future if a better approach is discovered/arises